Gemfile.lock

```diff
@@ -438,7 +438,7 @@ GEM
       rubocop-ast (>= 1.17.0, < 2.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 3.0)
-    rubocop-ast (1.17.0)
+    rubocop-ast (1.18.0)
```

Comment:
This is just linting the linter https://my.diffend.io/gems/rubocop-ast/1.17.0/1.18.0